contrib/kube-prometheus/README.md

```diff
@@ -2,221 +2,184 @@
-hack/cluster-monitoring/teardown
+```
+$ minikube delete && minikube start --kubernetes-version=v1.10.1 --memory=4096 --bootstrapper=kubeadm --extra-config=kubelet.authentication-token-webhook=true --extra-config=kubelet.authorization-mode=Webhook --extra-config=scheduler.address=0.0.0.0 --extra-config=controller-manager.address=0.0.0.0
 ```
 
-## Customizing
-
-As everyone's infrastructure is slightly different, different organizations have different requirements. Thereby there may be modifications you want to do on kube-prometheus to fit your needs.
-
-The kube-prometheus stack is intended to be a jsonnet library for organizations to consume and use in their own infrastructure repository. Below is an example how it can be used to deploy the stack properly on minikube.
-
-The three "distribution" examples we have assembled can be found in:
-
-* `example-dist/base`: contains the plain kube-prometheus stack for organizations to build on.
-* `example-dist/kubeadm`: contains the kube-prometheus stack with slight modifications to work properly monitoring kubeadm clusters and exposes UIs on NodePorts for demonstration purposes.
-* `example-dist/bootkube`: contains the kube-prometheus stack with slight modifications to work properly on clusters created with bootkube.
-
-The examples in `example-dist/` are purely meant for demonstration purposes, the `kube-prometheus.jsonnet` file should live in your organizations infrastructure repository and use the kube-prometheus library provided here.
-
-Examples of additoinal modifications you may want to make could be adding an `Ingress` object for each of the UIs, but the point of this is that as opposed to other solutions out there, this library does not need to yield all possible customization options, it's all up to the user to customize!
-
-### minikube kubeadm example
+## Quickstart
 
-See `example-dist/kubeadm` for an example for deploying on minikube, using the minikube kubeadm bootstrapper. The `example-dist/kubeadm/kube-prometheus.jsonnet` file renders the kube-prometheus manifests using jsonnet and then merges the result with kubeadm specifics, such as information on how to monitor kube-controller-manager and kube-scheduler as created by kubeadm. In addition for demonstration purposes, it converts the services selecting Prometheus, Alertmanager and Grafana to NodePort services.
+Although this project is intended to be used as a library, a compiled version of the Kubernetes manifests generated with this library is checked into this repository in order to try the content our quickly.
 
-Let's give that a try, and create a minikube cluster:
+Simply create the stack:
 
 ```
-minikube delete && minikube start --kubernetes-version=v1.9.6 --memory=4096 --bootstrapper=kubeadm --extra-config=kubelet.authentication-token-webhook=true --extra-config=kubelet.authorization-mode=Webhook --extra-config=scheduler.address=0.0.0.0 --extra-config=controller-manager.address=0.0.0.0
+$ kubectl create -f manifests/
```

Comment:
`/manifests` seems to miss the monitoring namespace manifest.

Reply:
do you think the namespace object should be part of the stack or we should just preceed this command with `kubectl create namespace monitoring`?

Reply:
To me the `monitoring` namespace definition should be part of the kube-prometheus project. I would prefer the declarative way via a manifest instead of the imperative way via `kubectl`.

Reply:
Added namespace object.

Reply:
Is there a dependency to run inside 'monitoring'?